tests/phpunit/tests/test-wpml-config-locations.php

```diff
@@ -3,29 +3,23 @@
 class WPML_Config_Locations_Test extends PLL_UnitTestCase {
 
 	protected static $dirs = array(
-		'themes' => array(
-			'best-theme',
-			'best-child',
-		),
-		'plugins' => array(
-			'best-plugin',
-		),
+		'themes/best-theme',
+		'themes/best-child',
+		'plugins/best-plugin',
 	);
 
 	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 		parent::wpSetUpBeforeClass( $factory );
 
 		// Copy themes and plugins from tests/phpunit/data to wp-content.
-		foreach ( self::$dirs as $type => $subdirs ) {
-			foreach ( $subdirs as $name ) {
-				$source_dir = dirname( __DIR__ ) . "/data/{$type}/{$name}";
-				$dest_dir   = WP_CONTENT_DIR . "/{$type}/{$name}";
+		foreach ( self::$dirs as $type => $path ) {
+			$source_dir = PLL_TEST_DATA_DIR . "/{$path}";
```

Comment:
`PLL_TEST_DATA_DIR` already has [a trailing slash](https://github.com/polylang/polylang/blob/236e8ba021211620557dfdd51e33beeef4a5be65/tests/phpunit/includes/bootstrap.php#L24).

```suggestion
			$source_dir = PLL_TEST_DATA_DIR . $path;
```